styles/theme.css

```diff
@@ -0,0 +1,77 @@
+div {
+  border: 1px solid black;

```

Comment:
Your borders are overlapping in certain places due to this line of css. You could look into removing the border in certain places. You can also look into the outline property